drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -469,7 +462,9 @@ function DragAndDropEditBlock(runtime, element, params) {
                             if (name.length > 0 || imageURL.length > 0) {
                                 var data = {
                                     displayName: name,
-                                    zone: $el.find('.zone-select').val(),
+                                    zones: $.map($el.find('.zone-checkbox:checked'), function(c){
```

Comment:
@haikuginger This is definitely readable but slightly terse. You could store the checkbox elements in a variable (like the code does for `name`, `imageURL`, etc.), and do `function(checkbox) {...}` instead of `function(c) {...}`.